translator/trace/zipkin/traces_to_zipkinv2_test.go

```diff
@@ -111,3 +117,37 @@ func TestInternalTracesToZipkinSpansAndBack(t *testing.T) {
 		assert.Equal(t, td.SpanCount(), tdFromZS.SpanCount())
 	}
 }
+
+func generateTraceTwoSpansOneNil() pdata.Traces {
```

Comment:
Is this not good as input for the test https://github.com/open-telemetry/opentelemetry-collector/blob/master/internal/data/testdata/trace.go#L169 ?

Reply:
You are right, I can use that, but firstly need to assign traceID to the first span from that pair, otherwise "No trace id" error shadows the panic